otx/mpa/modules/datasets/cls_dir_dataset.py

```diff
@@ -186,6 +198,9 @@ def evaluate(self, results, metric="accuracy", metric_options=None, logger=None)
             self.class_acc = True
 
         eval_results = super().evaluate(results, metrics, metric_options, logger)
+        for k in metric_options["topk"]:
```

Comment:
Do we need to update `otx/mpa/modules/datasets/cls_dir_dataset.py`? For my best knowledge, classification task is using `otx/algorithms/classification/adapters/mmcls/data/datasets.py`.

Reply:
@JihwanEom @kprokofi If we don't use this dataset, why do we need this file?

Reply:
Agree, maybe we can consider deleting all the unnecessary files?

Reply:
It will be removed soon; needed for MPA inner source repository. 
How about remove unnecessary files in other PR? We can clear unused files while writing unit tests.

Reply:
Yeah, in other PRs, of cource